tests/test_jams.py

```diff
@@ -101,8 +101,21 @@ def test_jobject_nonzero(data, value):
     assert J.__nonzero__() == value
 
 
-# Sandbox
+def test_jobject_repr():
+    repr(jams.JObject(foo=1, bar=2))
```

Comment:
hah, probably.  Actually none of these tests check output, just that the code executes properly.  I guess we could put some dummy checks in.